sunpy/map/map_factory.py

```diff
@@ -37,9 +37,6 @@
 except ImportError:
     pass
 
-__authors__ = ["Russell Hewett, Stuart Mumford"]
```

Comment:
I'm in support of removing the `__authors__` and `__email__` lists everywhere in the codebase, but we should probably ensure that those whose names appear there (i.e. mostly the early developers) are OK with it. This could be offloaded to a separate issue/PR (e.g. "Clean out old author info from the whole codebase")

Reply:
I *think* I remember @Cadair doing it as he went along, but may be mistaken. Would be +1 for doing it together in one PR.

Reply:
Doing a quick search on the code base shows that this is only 1/2 left so looks like this is a non-issue 😆

Reply:
Hmm, I wasn't aware of a desire to strip out all of the authors/emails from the codebase.  I'll point out that there are a bunch more if you search for `__author__` instead of `__authors__`, including my information a couple of times.

Reply:
Ah ok. That's my mistake. There clearly needs to be a larger discussion on this then. If people, particularly the early contributors such as yourself, would rather leave them in, I'm not opposed to that. My thinking was that we might as well remove them as we're now recording "authorship" through git.

Reply:
I created an issue (#3650) for discussion

Reply:
I've removed it all in my isort PR since its pointless.

Reply:
I disagree with the blanket "pointless", but that's a discussion to have on #3650

Reply:
+1 for removing this in this particular file btw, it's very wrong!